js/script.js

```diff
@@ -0,0 +1,11 @@
+const listingFilter = document.querySelector('.listing__filters');
+const btn = document.querySelector('.listing-header__right');
+
+
+btn.addEventListener('click', e => {
+    if(listingFilter.classList.contains('filter-list__open')) {
```

Comment:
The act of removing a class if it is already present and adding the class if not can be achieved by classList.toggle(<class-name>)

Makes the code much more crisp.